manifests/e/EclipseAdoptium/Temurin/11/jdk-11.0.14+9/EclipseAdoptium.Temurin.11.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: EclipseAdoptium.Temurin.11
+PackageVersion: jdk-11.0.14+9
```

needs better version matching
```suggestion
PackageVersion: 11.0.14.9
```